client/src/components/Navbar/Menu.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { useAuth } from '../../context/useAuthContext';
+import { AuthenticatedDesktop, AuthenticatedMobile, Unauthenticated } from '../Navbar/Menus';
+
+const Menu = (): JSX.Element => {
+  const { loggedInUser } = useAuth();
+  const [width, setWidth] = React.useState<number>(window.innerWidth);
+  const [mobile, setMobile] = React.useState<boolean>(width < 1025);
+
+  React.useEffect(() => {
+    const handleResize = () => {
+      setWidth(window.innerWidth);
+      setMobile(window.innerWidth < 1025 ? true : false);
+    };
+    window.addEventListener('resize', handleResize);
+    return () => window.removeEventListener('resize', handleResize);
+  }, [window]);
```

Comment:
This approach for handling resizing is a little bit hacky, it would probably be cleaner to just use breakpoints to conditionally display different versions of the Navbar

Reply:
Fixed in the upcoming commit. The new approach makes it so much easier, good call!